shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -328,6 +353,7 @@ class FlutterPlatformViewsController {
   fml::scoped_nsobject<FlutterMethodChannel> channel_;
   fml::scoped_nsobject<UIView> flutter_view_;
   fml::scoped_nsobject<UIViewController> flutter_view_controller_;
+  fml::scoped_nsobject<FlutterClippingMaskViewPool> mask_view_pool_;
```

Comment:
Is there a reason to make this a `scoped_nsobject`?

Reply:
No, I should change it to properties. I was blindly following the current pattern.

Reply:
Oh, actually, it is in a c++ class. So I think scoped_nsobject is better that the lifecycle is automatically managed.

Reply:
Which class is c++?  Your new `FlutterClippingMaskViewPool` is a Obj-C class.

Reply:
FlutterPlatformViewsController is the c++ class.